app/concerns/wpcc/salary_threshold.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  module SalaryThreshold
+    extend ActiveSupport::Concern
+
+    def load_config(file:)
+      YAML.load_file(Wpcc::Engine.root.join('config', "#{file}.yml"))
+    end
+  end
+end
```

Comment:
load_config seems like a 'helper' or 'utility' method that could/should be used by other classes that need to load a yml file, for example, the contribution_periods.yml file used by ContributionsCalendar (https://github.com/moneyadviceservice/wpcc/blob/master/app/models/wpcc/contributions_calendar.rb#L8-L9 can be refactored as a separate pr after this is merged).  Also could be used if we decide to store the session expiration period in a yml file. So it is not unique to SalaryThreshold.